packages/preview-service/src/jobProcessor.ts

```diff
@@ -98,13 +108,13 @@ const pageFunction = async ({
       case 'debug':
         logger.debug(msg.text())
       case 'error':
-        logger.warn({ err: msg }, 'Page error')
```

Comment:
@iainsproat just `{err: msg}` translates to an empty object, because the object isn't serializable